content/blog/2023/2023-q1-update/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Community update Q1 2023: Growing our community partner network and our team"
+subtitle: ""
+summary: ""
+authors: ["Chris Holdgraf"]
+tags: []
+categories: ["updates"]
+date: 2023-04-28
+featured: false
+draft: false
+---
+
+It's a month after the end of Q1 2023, and we'd like to share a belated update about what we were up to in the first quarter of this year (we have good excuses for being late, including new tiny humans, I promise).
+
+This quarter we grew our engineering team significantly, and started to refine our team processes and structures to accommodate this extra complexity.
+We expanded our managed cloud service with new community partners, and made a number of improvements to our technical infrastructure and organizational processes for managing this service.
+
+Read on below to learn more about what we've been up to!
+
+## New community partnerships
+
+We added several new community partnerships to our managed hub service.
+We've deployed new hub infrastructure for each of the following groups:
+
+- Our [University of Toronto Jupyter service](https://act.utoronto.ca/jupyterhub-support/) now has a dedicated R hub.
+- We ran an event hub for the [Drakkar Ocean project](https://www.drakkar-ocean.eu/)
+- We deployed a hub for the [NASA Visuaslization Exploration and Data Analysis (VEDA) project](https://www.earthdata.nasa.gov/esds/veda).
+- We deployed a hub for the [QuantifiedCarbon](https://www.linkedin.com/company/quantifiedcarbon/about/) organization.
+- We ran an event hub for [OceanHackWeek 2023](https://oceanhackweek.org/).
+- We onboarded several new community colleges to our "JupyterHubs Education in Community Colleges" collaboration with [CloudBank](https://www.cloudbank.org/welcome-cloudbank) and [UC Berkeley CDSS](https://data.berkeley.edu/).
+
+## Service improvements
+
+Below are a few highlights for ways in which we improved our [Managed Cloud Service](https://docs.2i2c.org) for our partner communities.
+
+### We simplified our authentication workflow with CILogon
+
+Authentication services allow us to identify a user when they log onto a hub, which determines their ability to access hub resources.
+Previously we had used a combination of [Auth0](https://auth0.com/) and [CILogon](https://www.cilogon.org/) for authentication.
```

Comment:
And GitHub as well (https://infrastructure.2i2c.org/hub-deployment-guide/configure-auth/github-orgs.html), but not sure if we need to mention it in this context.